content/kubeone/v1.5/references/terraform-integration/_index.en.md

```diff
@@ -99,6 +99,16 @@ output "kubeone_hosts" {
       bastion              = aws_instance.bastion.public_ip
       bastion_port         = var.bastion_port
       bastion_user         = var.bastion_user
+      bastion_host_key     = var.bastion_host_key
```

Comment:
This feature is available only starting with KubeOne 1.6 (to be released in February). Please drop this line from this file.

Reply:
Dropped, thanks!